Web/starter_project_barbies/pages/blog/BlogDetail.tsx

```diff
@@ -0,0 +1,137 @@
+import BlogCard from '@/components/blog/BlogCard'
+import fonts from '@/styles/blog/blog-detail.module.css'
+import { Blog } from '@/types/blog'
+import Image from 'next/image'
+
+interface BlogDetailProps {
+  blog: Blog
+}
+
+interface BlogAuthorDetailProps {
+  author: {
+    name: string
+    email: string
+    image: string
+    profession: string
+    userName: string
+  }
+}
+
+interface BlogContentProps {
+  content: string
```

Comment:
[nit] you can use something similar as above for this too
allowing us to change the type of the content if we want to (maybe we want to make it an HTML entity instead of a string or sth)

but to be honest i don' think we will change it